### PR TITLE
multiple pages: add korean translation

### PR DESCRIPTION
**Pull request by lch7167**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.ko/common/basename.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# basename
+
+> 경로명의 디렉토리가 아닌 부분을 반환.
+
+- 경로에서 파일 이름만 표시:
+
+`basename {{path/to/file}}`
```
Comment (sbrl) [inline pages.ko/common/basename.md]: Perhaps the generic ones here such as `path/to/file`, ` suffix`, `file1`, etc. could be translated too?

Comment (lch7167) [reply pages.ko/common/basename.md]: I think it's more natural. but if you want, i will translate it

File: pages.ko/common/behat.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# behat
+
+> Behaviour-Driven 개발을 위한 PHP 프레임워크. 
+> 자세한 정보: [https://behat.org](<https://behat.org/>).
```
Comment (einverne) [inline pages.ko/common/behat.md]:
```suggestion
> 자세한 정보: <https://behat.org/>.
```

File: pages.ko/common/blackfire.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# blackfire
+
+> PHP용 커맨드라인 프로파일링 도구. 
+> 자세한 정보: [https://blackfire.io](<https://blackfire.io/>).
```
Comment (einverne) [inline pages.ko/common/blackfire.md]:
```suggestion
> 자세한 정보: <https://blackfire.io/>.
```

File: pages.ko/common/bmaptool.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# bmaptool
+
+> 블록 맵을 지능적으로 생성 및 복사( `cp` 혹은 `dd`보다 빠른 속도).
+> More information: <https://source.tizen.org/documentation/reference/bmaptool>.
```
Comment (einverne) [inline pages.ko/common/bmaptool.md]: Should be `자세한 정보` here?

Comment (lch7167) [reply pages.ko/common/bmaptool.md]: @einverne Thanks for your comment. i fixed it

File: pages.ko/common/basename.md (hunk 1 above)
Comment (sbrl) [reply pages.ko/common/basename.md]: Hrm. I think we should translate them. For example, take a look at this `pt_BR` page: https://github.com/tldr-pages/tldr/blob/master/pages.pt_BR/common/7za.md

Basically, anything that doesn't affect the execution of the command should be translated.

Comment (lch7167) [reply pages.ko/common/basename.md]: @sbrl I translated them :)

**Pull request merged**